packages/standard-server-fetch/src/event-source.ts

```
@@ -0,0 +1,123 @@
+      }
+    }
+    finally {
+      await reader.cancel()
+    }
+  }
+
+  return gen()
+}
+
+export function toEventStream(
+  iterator: AsyncIterator<JsonValue | void, JsonValue | void, void>,
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)